adops/setting-up-prebid-video-in-dfp.md

```diff
@@ -38,29 +38,37 @@ Other line item settings and key/value targeting are the same as [those recommen
 + By default, Prebid.js caps all CPMs at $20.  As a video seller, you may expect to see CPMs higher than $20.  In order to receive those bids, you'll need to make sure your dev team implements custom price buckets as described in the [engineering setup instructions]({{site.baseurl}}/dev-docs/show-video-with-a-dfp-video-tag.html).  Once those changes are made on the engineering side, there should be no changes required from the ad ops side to support CPMs over $20.
 
 {: .alert.alert-success :}
-Be sure to duplicate your line item and video creative for each Prebid price bucket you intend to create.
+Be sure to duplicate your line item and video creative for each Prebid price bucket you intend to create. You may also need separate video line items for each cache service being used. e.g. if both AppNexus and Rubicon Project are bidders, you'll need separate line items to support the different cache URLs required.
```

Comment:
If Prebid Server cache is being used, can't a single cache URL be used across all bidders?

Reply:
In our case, we always cache the XML so don't need the client to do extra caching. In general, we prefer to rely on our infrastructure for delivering creatives.

The current status is that PBJS doesn't do the client-side caching if the `videoCacheKey` is in the bid response, so our video setup requires pubs to have separate creatives. We could provide an override for pubs who don't wish to do this.